src/components/NewTarget/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { View, Image } from 'react-native';
+import { Marker } from 'react-native-maps';
+import { shape, number, func } from 'prop-types';
+
+import targetMarker from 'assets/images/target-marker.png';
+import targetMarkerCircle from 'assets/images/target-marker-circle.png';
+
+import styles from './styles';
+
+const NewTarget = ({ coordinate, changeTargetPosition }) => {
+  return (
+    <Marker
+      coordinate={coordinate}
+      draggable
+      onDragEnd={changeTargetPosition}
+      onPress={e => e.stopPropagation()}>
+      <View style={styles.markerContainer}>
+        <Image source={targetMarkerCircle} style={styles.markerCircle} />
+        <Image source={targetMarker} style={styles.marker} />
+      </View>
+    </Marker>
+  );
+};
```

Comment:
you can use implicit return here
```suggestion
const NewTarget = ({ coordinate, changeTargetPosition }) => (
    <Marker
      coordinate={coordinate}
      draggable
      onDragEnd={changeTargetPosition}
      onPress={e => e.stopPropagation()}>
      <View style={styles.markerContainer}>
        <Image source={targetMarkerCircle} style={styles.markerCircle} />
        <Image source={targetMarker} style={styles.marker} />
      </View>
    </Marker>
  );
```

Reply:
also try to avoid the boolean `dragabble` in the middle of the props, maybe on top or bottom <Marker dragabble coordinates > 
```suggestion
const NewTarget = ({ coordinate, changeTargetPosition }) => {
  return (
    <Marker
      draggable
      coordinate={coordinate}
      onDragEnd={changeTargetPosition}
      onPress={e => e.stopPropagation()}>
      <View style={styles.markerContainer}>
        <Image source={targetMarkerCircle} style={styles.markerCircle} />
        <Image source={targetMarker} style={styles.marker} />
      </View>
    </Marker>
  );
};
```